dev/integration_tests/android_views/android/app/src/main/java/io/flutter/integration/androidviews/SimplePlatformView.java

```diff
@@ -43,14 +45,34 @@ public void dispose() {
     public void onMethodCall(MethodCall methodCall, MethodChannel.Result result) {
         switch(methodCall.method) {
             case "pipeTouchEvents":
-                mTouchPipe.enable();
+                touchPipe.enable();
                 result.success(null);
                 return;
             case "stopTouchEvents":
-                mTouchPipe.disable();
+                touchPipe.disable();
                 result.success(null);
                 return;
+            case "showAndHideAlertDialog":
+                showAndHideAlertDialog(result);
+                return;
         }
         result.notImplemented();
     }
+
+    private void showAndHideAlertDialog(MethodChannel.Result result) {
+        Context context = view.getContext();
+        AlertDialog.Builder builder = new AlertDialog.Builder(context);
+        TextView textView = new TextView(context);
+        textView.setText("This alert dialog will close in 1 seconds");
+        builder.setView(textView);
+        final AlertDialog alertDialog = builder.show();
+        result.success(null);
+        view.postDelayed(new Runnable() {
```

Comment:
It feels a bit unreliable for the alert to hide after a certain time. Maybe we should make a `hideAlertDialog` method to hide the dialog manually.

Reply:
It's a non normal flow to tap a button that is below the alert dialog with the driver extension... (as normally clicks won't go through it).

I'm going to just completely remove this timeout.